--- original/functions/button_callbacks.lua	Tue Aug 27 03:54:22 2024
+++ modded/functions/button_callbacks.lua	Tue Aug 27 18:26:31 2024
@@ -909,7 +909,7 @@
   end
 
   local OSkeyboard_e = e.parent.parent.parent
-  if G.CONTROLLER.text_input_hook == e and G.CONTROLLER.HID.controller then
+  if G.CONTROLLER.text_input_hook == e and (G.CONTROLLER.HID.controller or G.CONTROLLER.HID.touch) then
     if not OSkeyboard_e.children.controller_keyboard then 
       OSkeyboard_e.children.controller_keyboard = UIBox{
         definition = create_keyboard_input{backspace_key = true, return_key = true, space_key = false},
