--- original/resources/shaders/negative.fs	Fri Jan 26 19:02:59 2024
+++ modded/resources/shaders/negative.fs	Sun Aug 11 15:32:52 2024
@@ -4,4 +4,7 @@
 	#define MY_HIGHP_OR_MEDIUMP mediump
 #endif
+#ifdef GL_ES
+	precision MY_HIGHP_OR_MEDIUMP float;
+#endif
 
 extern MY_HIGHP_OR_MEDIUMP vec2 negative;
@@ -95,5 +98,5 @@
 }
 
-vec4 effect( vec4 colour, Image texture, vec2 texture_coords, vec2 screen_coords )
+mediump vec4 effect( mediump vec4 colour, Image texture, mediump vec2 texture_coords, mediump vec2 screen_coords )
 {
     vec4 tex = Texel(texture, texture_coords);
